docs/konflux-onboarding.md

```diff
@@ -70,7 +70,11 @@ FBC components and their validation are uniquely different from that of normal c
 
 ### Create the FBC in the git repository
 
-If you already have a graph defined in a previous index, it is recommended to migrate this graph to a `basic` FBC template
+**NOTE:** You must use OPM [v1.47.0](https://github.com/operator-framework/operator-registry/releases/tag/v1.47.0) or higher to run these commands.
```

Comment:
Its 1.46.0, right?

Reply:
I tried with 1.46 and it didn't work for me; also why not tell people to use the latest?

Reply:
We can recommend the latest, but the version will easily be out of date if what we really want is `latest`.

I don't know why it didn't work for you with 1.46.0. I have no problem with suggesting at least 1.47 then.